format/FlightSql.proto

```diff
@@ -55,7 +55,7 @@ message CommandGetSqlInfo {
    * Initially, Flight SQL will support the following information types:
    * - Server Information - Range [0-500)
    * - Syntax Information - Range [500-1000)
-   * Range [0-10,000) is reserved for defaults (see SqlInfo enum for default options). 
+   * Range [0-10,000] is reserved for defaults (see SqlInfo enum for default options).
```

Comment:
This seems wrong if custom options start at 10_000.

Reply:
Yeah, this was an error. Reverting this change